imports/style/components/_employee.scss

```diff
@@ -0,0 +1,17 @@
+.component--employee__employee-list{
+	height: 100vh;
+	width: 100vw;
+	overflow:scroll; 
+	position:relative;
+	.element--input__normal{
```

Comment:
should we have a corresponding `.element--input__large` here to set the style for the large input?

Reply:
I think this is the default normal size, we can apply large when we need to change it.